### DISCUSSION TITLE
Stuck in Ultimate with a broken display

### DISCUSSION THREAD

**Discussion by Nundahl** [seerge/g-helper#1356] [Q&A]
Hi! My ASUS ROG Flow Z13 internal display has failed in such a way that I cannot see GHelper enough to switch back to Standard mode and to allow me to use my EGPU or otherwise an external display to access the device. I'm hoping there might be some way to reset that or switch GPU modes/reconnect to my XG Mobile on with key commands possibly.

Anything that could get me to an external display so I can move some data off before attempting repair by technician would be great.

Open to any and all suggestions! Thanks to the community and the developers here for a great product.
![20230923_204245](https://github.com/seerge/g-helper/assets/145886809/4b3a66b5-0198-4c4f-bf48-05d2dc6eb310)

**Comment by seerge** [comment]
@Nundahl hello, you can try [hardware reset ](https://github.com/seerge/g-helper#how-do-i-do-a-hardware-reset-on-a-laptop) - it would set device back to standard mode

**Reply by Nundahl** [reply]
I should have said but I have tried this, no luck unfortunately. I'll give it another shot to make sure I did it properly, thanks for the response!

**Reply by seerge** [reply]
@Nundahl alternatively boot to BIOS (F2 on boot). Does the screen work for you in that case? If so - there should be an option to swtich between dedicated GPU and standard as well.

**Reply by Nundahl** [reply]
Unfortunately the display is broken fully so no good in bios too, so what I need is some way to get it to display on a secondary monitor. But I'll do some trying.

**Reply by Nundahl** [reply]
Confirmed, no luck with a reset or bios. Hard to know if the reset is actually taking or not because I can't see but for sure followed the instructions just to double check myself.

I think at this point my best bet will be pulling the hard drive and accessing it from another system.
